panel/models/echarts.ts

```diff
@@ -89,6 +89,11 @@ export class EChartsView extends HTMLBoxView {
     if ((window as any).echarts == null) {
       return
     }
+    const series = this.model.data.series[0]
+    if (typeof series.axisLabel.formatter === "string") {
```

Comment:
Its a general problem with our echarts implementation that we cannot use a js function. Often you find some example in the documentation. But it uses one or more js functions and we cannot use that.

Reply:
See pyecharts might even support/ work with python functions.

https://github.com/pyecharts/pyecharts-javascripthon

Reply:
PyEcharts provides `JSCode` utility class for supporting js functions.

https://github.com/pyecharts/pyecharts/blob/5228492426e7f7bd6d701af8ae1b130d5c14145c/pyecharts/commons/utils.py#L6

Reply:
> Its a general problem with our echarts implementation that we cannot use a js function

I will go even further, as each of these escape hatches with static configurations has this problem, another example is [tabulator](https://panel.holoviz.org/reference/widgets/Tabulator.html#static-configuration). 

So I agree we should have a general solution for this. I need to look into how this can be done, as it needs to be done much earlier than here, likely with a custom model. 

This was mainly a POC I pushed (and still a draft) to check if I could do it for this requested instance.